defaults/main.yml

```diff
@@ -70,12 +70,24 @@ neofs_storage__notification_timeout: '6s'
 
 neofs_storage__tls_enabled: False
 neofs_storage__tls_endpoint: "{{ neofs_storage__grpc_address }}:{{ neofs_storage__grpcs_port }}"
-# Local paths to TLS certificate and key
-neofs_storage__tls_cert_path: "/path/to/cert"
-neofs_storage__tls_key_path: "/path/to/key"
-# Remote paths to TLS certificate and key
-neofs_storage__tls_cert: "{{ neofs_storage__conf_dir }}/tls.cert"
-neofs_storage__tls_key: "{{ neofs_storage__conf_dir }}/tls.key"
+# TLS certificate and key content
+neofs_storage__tls_cert: ''
+neofs_storage__tls_key: ''
+# Remote files
+neofs_storage__tls_cert_file: "{{ neofs_storage__conf_dir }}/tls.cert"
+neofs_storage__tls_key_file: "{{ neofs_storage__conf_dir }}/tls.key"
+
+neofs_storage__grpc_endpoints:
+  - address: "{{ neofs_storage__grpc_address }}"
```

Comment:
Same binding point twice. Will it work?

Reply:
ports are different: `neofs_storage__grpc_port` and `neofs_storage__grpcs_port`